resources/js/src/app/mixins/baseDropdown.js

```diff
@@ -0,0 +1,36 @@
+export default {
+    props: [
+        "template",
+        "facet"
+    ],
+
+    data: function () {
+        return {
+            isShowDropdown: false
```

Comment:
I know this was here before, but `isShowDropdown` is a pretty odd name. How about renaming it to just `showDropdown` or maybe even just `isOpen`/`open`, since we're already in the context of a dropdown.